src/styles/blocks.scss/card.scss

```diff
@@ -0,0 +1,105 @@
+.card {
+  font-family: Roboto, sans-serif;
+  box-sizing: border-box;
+  width: 200px;
+  padding: 16px;
```

Comment:
Consider using variables for values that are reused, like this padding value. This will make your code more maintainable and easier to read. For example, you could define a $default-padding variable in your variables.scss file and use it here.